src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,69 @@
+.nav{
+  display: flex;
+  align-items: center;
+  height: 100%;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
```

Comment:
You can reset all default styles in `index.scss` or make a separate `reset.scss` file
```suggestion
```